core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -67,6 +68,10 @@ var ExplorationPlayerPage = function() {
     viewSolutionButton.click();
   };
 
+  this.gotIt = function() {
```

Comment:
Please change the name to this.clickGotItButton() -- the method name should start with a verb indicating the action (which is "click").

Reply:
Done.